py/selenium/webdriver/remote/webdriver.py

```diff
@@ -82,11 +87,11 @@ def __init__(self, command_executor='http://127.0.0.1:4444/wd/hub',
         if type(self.command_executor) is bytes or isinstance(self.command_executor, str):
             self.command_executor = RemoteConnection(command_executor, keep_alive=keep_alive)
         self._is_remote = True
-        self.session_id = None
-        self.capabilities = {}
+        self.session_id = session_id
+        self.capabilities = capabilities
         self.error_handler = ErrorHandler()
         self.start_client()
-        self.start_session(desired_capabilities, browser_profile)
+        self.start_session(desired_capabilities, browser_profile, force_new=False)
```

Comment:
the 'force_new' variable doesn't make sense to default to true in the method, but the only place calling this (mostly private method, i know it's "public" and doc'd, but the constructor calls it) is here and it sets it to false. Seems like you're leaking a bit of how you use webdriver into here. Because it doesn't make sense to even have it, either you pass a sessionId to the constructor or you don't.

Or please show me some sample code where this makes sense and maybe we'll need to add that to the doc for the start_session method.